### PR TITLE
Quiet option removed

### PR DESCRIPTION
**Pull request by pranita-s**
Quiet option is removed. Also, before returning bone variable, its length is being checked, if its zero then it implies the dataset was not found and hence it need not be returned. Not sure if anything else has to be changed.
This solves #133 and #113. @ethanwhite @henrykironde Please review this PR.

### CODE REVIEW

File: R/rdataretriever.R [hunk 1]
```diff
@@ -136,9 +137,11 @@ fetch = function(dataset, quiet=TRUE, data_names=NULL){
         }
         bone[[i]] = tempdata
     }
+    if(length(bone)>0){
     if (length(bone) == 1)
```
Comment (henrykironde) [inline R/rdataretriever.R]: Come code refactoring/indentation may be needed here

File: R/rdataretriever.R [hunk 2]
```diff
@@ -103,7 +103,8 @@ install = function(dataset, connection, db_file=NULL, conn_file=NULL,
 #' names(vegdata$plant_comp_ok)
 #' }
 fetch = function(dataset, quiet=TRUE, data_names=NULL){
-    temp_path = tempdir()
+    temp_path = tolower(tempdir())
```
Comment (henrykironde) [inline R/rdataretriever.R]: If we use `temp_path = tolower(tempdir())`, this looks like we create a directory say `./myTempDir_xyz` and we then use the lower case `. ./mytempdir_xyz`. Let me know if I have it wrong

File: R/rdataretriever.R (hunk 1 above)
Comment (pranita-s) [reply R/rdataretriever.R]: Sure ! I will indent it properly

File: R/rdataretriever.R (hunk 2 above)
Comment (pranita-s) [reply R/rdataretriever.R]: Yes, it is doing exactly as you said. Please see #132.

**Comment by ethanwhite** [comment]
@henrykironde - for this PR we're basically just reviewing 575c951d505117fa52ba81e0ebc1e3d2f2e06080 & 4c2ac6021795e5147e7e461a7b76b8a42e01c66c since the other changes are already merged (not sure why they are showing up here).

**Comment by pranita-s** [comment]
@ethanwhite Sure!

**Comment by pranita-s** [comment]
@ethanwhite @henrykironde I have made required changes to fetch function in the newest commit. It functions as below- 

nameddataset

Also, I have suppressed the warning of `(dir.create(temp_path)) `. If fetch function is run multiple times in one session, it displays a warning temp directory already exists. I suppressed it because I do not think its crucial.
Please let me know what you think about it.

File: R/rdataretriever.R [hunk 3]
```diff
@@ -103,42 +103,39 @@ install = function(dataset, connection, db_file=NULL, conn_file=NULL,
...
+        else ((length(data_names) == 1) & (length(datasets) == 1))
             stop("Assign name through the output instead (e.g., yourname = fetch('dataset')")
         names(bone) = data_names
     }
     for (i in seq_along(dataset)) {
         if (quiet)
-            run_cli(paste('retriever -q install csv --table_name',
-                          file.path(temp_path, '{db}_{table}.csv'),
-                          dataset[i]))
+            run_cli(paste('retriever install csv --table_name',file.path(temp_path, '{db}_{table}.csv'),dataset[i]))
```
[the remark this answers is not in the document]
Comment (pranita-s) [reply R/rdataretriever.R]: @ethanwhite Sure ! I will take care of the these points ! And next time I will make sure that I have understood the issue correctly.

**Comment by ethanwhite** [comment]
Thanks @pranita-s! This is great!

**Pull request merged**